packages/clay-tabs/src/__tests__/index.tsx

```diff
@@ -0,0 +1,175 @@
+				</ClayTabs.TabPane>
+			</ClayTabs.Content>
+		</>
+	);
+};
+
+describe('ClayTabs', () => {
+	afterEach(cleanup);
+
+	it('renders', () => {
+		const {container} = render(<ClayTabs />);
+
+		expect(container).toMatchSnapshot();
+	});
+	it('renders justifying nav items', () => {
+		const {container} = render(<ClayTabs justified />);
+
+		expect(container).toMatchSnapshot();
+	});
+	it('renders with modern style', () => {
+		const {container} = render(<ClayTabs modern />);
+
+		expect(container).toMatchSnapshot();
+	});
+	it('renders with items', () => {
+		const {container} = render(<ClayTabsWithItems />);
+
+		expect(container).toMatchSnapshot();
+	});
+	it('renders nav items as anchors', () => {
+		const {container, getByTestId} = render(
+			<>
+				<ClayTabs>
+					<ClayTabs.Item
+						active
+						component="anchor"
+						tabName="One"
+						tabkey={1}
+					/>
+					<ClayTabs.Item
+						component="anchor"
+						tabName="Two"
+						tabkey={2}
+					/>
+				</ClayTabs>
+				<ClayTabs.Content activeTabKey={1}>
+					<ClayTabs.TabPane tabkey={1}>
+						{'Content One'}
+					</ClayTabs.TabPane>
+					<ClayTabs.TabPane tabkey={2}>
+						{'Content Two'}
+					</ClayTabs.TabPane>
+				</ClayTabs.Content>
+			</>
+		);
+
+		const tabItem = getByTestId('oneTabItem');
+
+		expect(tabItem.nodeName).toBe('A');
+
+		expect(container).toMatchSnapshot();
+	});
+	it('renders disabled nav items', () => {
+		const {container} = render(
+			<>
+				<ClayTabs>
+					<ClayTabs.Item active tabName="One" tabkey={1} />
+					<ClayTabs.Item tabName="Two" tabkey={2} />
+					<ClayTabs.Item disabled tabName="Three" tabkey={3} />
+				</ClayTabs>
+				<ClayTabs.Content activeTabKey={1}>
+					<ClayTabs.TabPane tabkey={1}>
+						{'Content One'}
+					</ClayTabs.TabPane>
+					<ClayTabs.TabPane tabkey={2}>
+						{'Content Two'}
+					</ClayTabs.TabPane>
+					<ClayTabs.TabPane tabkey={3}>
+						{'Content Three'}
+					</ClayTabs.TabPane>
+				</ClayTabs.Content>
+			</>
+		);
+
+		expect(container).toMatchSnapshot();
+	});
+	it('emits a number when clicking an item', () => {
+		const onClick = jest.fn();
+
+		const {container, getByTestId} = render(
+			<>
+				<ClayTabs>
+					<ClayTabs.Item
+						active
+						onClick={onClick}
+						tabName="One"
+						tabkey={1}
+					/>
+					<ClayTabs.Item
+						component="anchor"
+						onClick={onClick}
+						tabName="Two"
+						tabkey={2}
+					/>
+				</ClayTabs>
+				<ClayTabs.Content activeTabKey={1}>
+					<ClayTabs.TabPane tabkey={1}>
+						{'Content One'}
+					</ClayTabs.TabPane>
+					<ClayTabs.TabPane tabkey={2}>
+						{'Content Two'}
+					</ClayTabs.TabPane>
+				</ClayTabs.Content>
+			</>
+		);
+
+		fireEvent.click(getByTestId('oneTabItem'));
+		expect(onClick).toBeCalled();
+		expect(onClick).toBeCalledWith(1);
+
+		fireEvent.click(getByTestId('twoTabItem'));
+		expect(onClick).toBeCalled();
+		expect(onClick).toBeCalledWith(2);
+
+		expect(container).toMatchSnapshot();
```

Comment:
It is necessary to run a `toMatchSnapshot` here, we will try to minimize the use of snapshot in our tests.

Reply:
Nope. Removing.